inference-engine/thirdparty/clDNN/tests/test_cases/reorder_gpu_test.cpp

```diff
@@ -2394,3 +2394,57 @@ INSTANTIATE_TEST_SUITE_P(DISABLED_REORDER,
                         reorder_test,
                         ::testing::ValuesIn(reorder_test::generate_specific_test_params()),
                         tests::generic_test::custom_param_name_functor());
+
+
+#ifdef ENABLE_ONEDNN_FOR_GPU
+TEST(reorder_onednn_gpu, basic_convert_int8) {
+
+    auto& engine = get_onednn_test_engine();
+    layout in_layout = { type_to_data_type<float>::value,format::byxf,{ 1, 1, 3, 3 } };
+    layout byte_layout = { type_to_data_type<int8_t>::value, format::bfyx,{ 1, 1, 3, 3 } };
+    std::initializer_list<float> input_f = { 1.0f, -2.6f, 3.1f, -4.0f, 5.03f, -6.99f, 7.0f, -8.0f, 9.0f };
+    std::list<float> final_results = { 1.0f, -3.0f, 3.0f, -4.0f, 5.0f, -7.0f, 7.0f, -8.0f, 9.0f };
+
+    // Allocate memory for input image.
+    auto input_memory = engine.allocate_memory(in_layout);
+    set_values(input_memory, input_f);
+
+    // Create input_layout description
+    // "input" - is the primitive id inside topology
+    input_layout input("input", in_layout);
+
+    topology topology(
+        // 1. input layout primitive.
+        input,
+        // 2. reorder primitive with id "reorder_input"
+        reorder("reorder_input",
+            // input primitive for reorder (implicitly converted to primitive_id)
+            input,
+            // output layout for reorder
+            byte_layout),
+        reorder("reorder2", "reorder_input", in_layout)
+    );
+
+    build_options options_target;
+    options_target.set_option(build_option::outputs({ "reorder_input", "reorder2"}));
+    implementation_desc impl = {format::bfyx, std::string(""), impl_types::onednn};
+    options_target.set_option(build_option::force_implementations({{"reorder_input", impl}}));
+
+    network network(
+        engine,
+        topology,
+        options_target);
+
+    network.set_input_data("input", input_memory);
+
+    auto outputs = network.execute();
+
+    auto interm = outputs.at("reorder2").get_memory();
+    cldnn::mem_lock<float> interm_ptr(interm, get_test_stream());
+    unsigned int cntr = 0;
+    for (const auto& exp : final_results)
+    {
+        EXPECT_EQ(exp, interm_ptr[cntr++]);
+    }
+}
+#endif
```

Comment:
Please add new line to the end of the file

Reply:
applied, thanks.